### ISSUE TITLE
Crash systemUI on boot

### ISSUE THREAD

**Issue by Underday** [siavash79/AOSPMods#95]
Just flashed latest version, on first boot (And yeah ı tried reboot twice)
But everything is black. 

 - Device: Poco X3 Nfc
 - OS: CrDroid 12.1

**Comment by Underday** [comment]
I tried different version but result is same. There is no conflict module ı think, and no weird system modification.
I'm using Lsposed Zygisk version (v1.8.2 ı think it is latest). I looked in the Magisk and Lsposed log but can't see any problematic log. (After delete module in twrp, of course because ı cant access anything without delete)

**Comment by siavash79** [comment]
So do you mean that you got bootloop?

Can you please share the Lsposed log zip file?

**Comment by Underday** [comment]
Not bootloop, it's opening but system UI isn't opening or something.
And here is lsposed log. 
[LSPosed_2022-04-27T10_37_35.824.zip](https://github.com/siavash79/AOSPMods/files/8569935/LSPosed_2022-04-27T10_37_35.824.zip)

**Comment by Indra1402-sfyn** [comment]
I'm having the same problem. After I reboot the device

**Comment by siavash79** [comment]
From what I can see, you've tried the canary 10 version.

Can you please try again with canary 12 and let me know if successful or not, and share lsposed log?
One more request: can you try without any other mod active in lsposed (temporarily)? (referring to 2 other mods installed on your device)

**Comment by Underday** [comment]
sorry i just changed rom.It works completely fine in this rom, maybe different logs from the same device will be of use to you. Yes, I am using two more mods, but it still works with the same mods. I am using the same magisk modules too.

At first I tried version 12, it didn't work so I tried version 10 again. 

As you said in telegram I just tried installing the app instead of the magisk module.And still the same problem happened, I solved the problem by disable root rights. 

And here is new logs:
[LSPosed_2022-04-27T23_31_59.263.zip](https://github.com/siavash79/AOSPMods/files/8577566/LSPosed_2022-04-27T23_31_59.263.zip)

Arcana rom Android 12.1 (Poco x3)

Finally, I tried with a few kernels because I thought it was kernel sourced, but the error still same.
